src/getParams.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const getParams = (url) => {
+  const [path, inputSearchParams] = url.split('?');
+
+  const pathParams = path.slice(1).split('/');
+
+  const searchParams = new URLSearchParams(inputSearchParams);
+  const searchParamsNames = inputSearchParams
+    .split('&')
+    .map(searchParam => searchParam.split('=')[0]);
```

Comment:
A simpler way to do this, [doc](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/keys). It will probably fix your error too :)
```suggestion
  const searchParamsNames = searchParams.keys();
```